sklearn/utils/linear_assignment_.py

```diff
@@ -115,6 +119,11 @@ def _hungarian(cost_matrix):
         The pairs of (row, col) indices in the original array giving
         the original ordering.
     """
+    warnings.warn(
+        "This method is deprecated in 0.21 and will removed from 0.23 "
```

Comment:
Maybe indicate the function name and what to use instead:

> The linear_assignment function is deprecated [...] Use scipy.optimize.linear_sum_assignment instead

Reply:
done